Felienne/hedy
# [BUG FIX] Parsons exercises with indentation work as expected

**Description**
In this PR we fix the parsons exercises with indentation that didn't work as expected yet. We fix this by changing the YAML structure to make sure the whitespaces are preserved. We only fix this for the English YAML and the translated NL exercises as the non-translated ones should automatically be overwritten by Weblate.
![Schermafbeelding 2022-06-23 om 13 43 01](https://user-images.githubusercontent.com/38155424/175291155-890d1c70-ac12-43b6-ad60-18c004912958.png)

**Fixes**
This PR fixes #2939.

**How to test**
Navigate to level 8 and verify that the parsons exercise with indentation is working as expected.
